src/components/Kv/KvResponsiveImage2.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+	<!-- PICTURE ELEMENT RESPONSIVE IMAGE POC -->
+	<!-- Send picture element and srcset to to get eager image fetching -->
+	<!-- https://developers.google.com/web/fundamentals/design-and-ux/responsive/images -->
+	<!-- https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images -->
+	<!-- https://dev.to/stefanoverna/how-to-offer-responsive-progressive-images-in-2020-in-one-line-5fed -->
+
+	<picture>
+		<!-- browser uses the first <source> tag matching media query even if later <source> tags also match -->
+		<!-- So start with the largest window sizes first -->
+		<source
+			v-for="image in standardImages.reverse()"
+			:key="image[1]"
+			:media="`(min-width: ${getPxSizeForImage(image[0])}px)`"
+			:srcset="getSrcsetForImage(image)"
+		>
+		<img
+			:src="standardImages[0][1]"
+			v-bind="$attrs"
+			ref="img"
+		>
+	</picture>
+</template>
+
+<script>
+import _map from 'lodash/map';
+import _throttle from 'lodash/throttle';
+import { isHighDensity, isRetina } from '@/util/checkScreenDensity';
+
+export default {
+	props: {
+		// An array of image sizes and urls, e.g. [['small', 'img-sm.jpg'], ['small retina', 'img-sm-retina.jpg']]
+		images: {
+			type: Array,
+			required: true
+		}
+	},
+	data() {
+	// initializing isRetina as null and using the v-show above prevents flashing of a
```

Comment:
screen-size-mixin.js is merged into this component since that's the only place it's currently used.